numpy_financial/_financial.py

```diff
@@ -511,33 +513,29 @@ def ppmt(rate, per, nper, pv, fv=0, when='end'):
     return total - ipmt(rate, per, nper, pv, fv, when)
 
 
-def pv(rate, nper, pmt, fv=0, when='end'):
+def pv(
+    rate: Union[int, float, Decimal, np.ndarray],
```

Comment:
I added type hints here, but I see that they're not in the whole repo and sometimes supplied in the doc strings. NP to remove the type hints if preferred for consistency.

Using `Union` since the package supports Python 3.9. Starting in Python 3.10, we can do `int|float|Decimal|np.ndarray`.

Reply:
Type hints are great. Adding type hints is actually part of what I'll be working on once I have the broadcasting done.